h/services/bulk_annotation.py

```diff
@@ -58,6 +58,7 @@ def date_match(column: sa.Column, spec: dict):
 class BulkAnnotation:
     username: str
     authority_provided_id: str
+    metadata: dict
```

Comment:
We could chose another name here (and in the view return values). 

I reckon being consistent with the same concept in the models is better than introducing more confusion trying to name it "extra" or similar here.

Reply:
Ok, so this is `BulkAnnotationService`'s local `BulkAnnotation` dataclass that it uses for the objects that it returns, and we're adding a `metadata` dict to it :+1: